src/api/todos.ts

```diff
@@ -0,0 +1,20 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const USER_ID = 2158;
+
+export const getTodos = () => {
+  return client.get<Todo[]>(`/todos?userId=${USER_ID}`);
```

Comment:
let's move `/todos` into const and reuse

Reply:
already saved in state: `const [todos, setTodos] = useState<Todo[]>([]);`